test.py

```diff
@@ -5,24 +5,34 @@
 
 callCount = 0
 
+
 @app.get("/")
 async def h():
     global callCount
-    callCount +=  1
+    callCount += 1
     message = "Called " + str(callCount) + " times"
     return message
 
+
+@app.post("/post")
+async def postreq(body):
+    return bytearray(body).decode("utf-8")
```

Comment:
Are you using `bytearray` because we do not support json dicts as of now?

Reply:
I exported bytes as, raw bytes uploaded is a valid body of a request, if it is JSON they're expecting they can serialise it themselves.